deploy-manage/security/self-tls.md

```diff
@@ -0,0 +1,34 @@
+---
+navigation_title: "Self-managed"
+applies_to:
+  deployment:
+    self: ga
+---
+
+% scope: TLS encryption tasks and settings after the initial setup is completed.
+# Manage TLS encryption in self-managed deployments
+
+This section provides guidance on managing TLS certificates in self-managed deployments after the initial security setup. It covers tasks such as configuring mutual authentication, renewing certificates, and customizing supported TLS versions and cipher suites.
+
+::::{note}
+If you're looking to secure a new or existing cluster by setting up TLS for the first time, refer to [](./self-setup.md), which covers both the [automatic](./self-auto-setup.md) and [manual](./self-setup.md#manual-configuration) configuration procedures.
+::::
+
+The topics in this section focus on post-setup tasks:
+
+* [](./kibana-es-mutual-tls.md) (**optional**): Strengthen security by requiring {{kib}} to use a client certificate when connecting to {{es}}.
```

Comment:
are any of these required? would skip the `optional`

Reply:
Totally, good catch! it's a leftover.